modules/api/src/main/java/org/apache/fluo/api/config/FluoConfiguration.java

```diff
@@ -329,24 +332,41 @@ private void verifyApplicationName(String name) {
           "Invalid application name \"" + name + "\" caused by " + reason);
     }
   }
-
+  /**
+   * Sets the Zookeeper instance.
+   * @param zookeepers The instance to use, must not be null.
+   *
+   */
   public FluoConfiguration setInstanceZookeepers(String zookeepers) {
     return setNonEmptyString(CONNECTION_ZOOKEEPERS_PROP, zookeepers);
   }
-
+  
+  /**
+   * Returns the Zookeeper instance.
+   * @return The zookeeper instance.
+   */
   public String getInstanceZookeepers() {
     return getDepNonEmptyString(CONNECTION_ZOOKEEPERS_PROP, CLIENT_ZOOKEEPER_CONNECT_PROP,
         CONNECTION_ZOOKEEPERS_DEFAULT);
   }
-
+  /**
+   * Returns the zookeeper application name string.
+   * @return The zookeeper application string.
+   */
   public String getAppZookeepers() {
     return getInstanceZookeepers() + "/" + getApplicationName();
   }
-
+  /**
+   * Set the zookeeper timeout property. 
+   * @param timeout Must be a positive integer.
+   */
   public FluoConfiguration setZookeeperTimeout(int timeout) {
     return setPositiveInt(CONNECTION_ZOOKEEPER_TIMEOUT_PROP, timeout);
   }
-
+  /**
+   * Returns the custom zookeeper timeout if set, default otherwise.
```

Comment:
The following returns a default value if not set, so the docs could mention this.  Also in this case `CLIENT_ZOOKEEPER_TIMEOUT_PROP` is deprecated and does not need to be mentioned.

```
<p>
 Gets the value of the property {@value #CONNECTION_ZOOKEEPER_TIMEOUT_PROP} and if not set returns the default  {@value#CONNECTION_ZOOKEEPER_TIMEOUT_DEFAULT}
```